index.css

```diff
@@ -381,3 +399,85 @@ form textarea {
   font-size: 15px;
   color: #172b4d;
 }
+
+@media (min-width: 768px) {
+  nav .menu-bar {
+    display: none;
+  }
+
+  nav {
+    justify-content: space-around;
+    gap: 240px;
+  }
+
+  nav .drop-down {
+    display: inline-block;
+    position: inherit;
+  }
+
+  .drop-down ul {
+    display: flex;
+    justify-content: space-between;
+    gap: 40px;
+  }
+
+  .drop-down ul li a {
+    color: #344563;
+  }
+
+  .headline-content {
+    width: 65%;
+  }
+
+  .headline-content h1 span::after {
+    content: "\a";
+    white-space: pre;
+  }
+
+  .project-item:nth-of-type(1),
+  .project-item:nth-of-type(3) {
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .project-item:nth-of-type(2),
+  .project-item:nth-of-type(4) {
+    flex-direction: row-reverse;
+    gap: 20px;
+  }
+
+  .project-image {
+    width: 100%;
+  }
+
+  .project-details {
+    width: 100%;
+  }
+
+  .about-myself-section {
+    flex-direction: row;
+    justify-content: space-around;
+  }
+
+  .about-myself-description {
+    width: 100%;
+    margin-left: 5%;
+  }
+
+  .about-myself-skills {
+    width: 100%;
+    margin-right: 5%;
+  }
+
+  .contact-description {
+    margin-inline: 32%;
+  }
+
```

Comment:
- Please also note that the background images in your contact section does not exactly match what is in the figma template, please try to fix that 🙏

Your project looks like this:
![contact figma 1 temlate](https://user-images.githubusercontent.com/73607512/192862962-98e688e4-1c94-4d74-a67c-b758fdb9e278.PNG)

Figma template-
![figma desktop contact](https://user-images.githubusercontent.com/73607512/192863607-bd1018bd-c6c8-468f-a625-4f64df2183c9.PNG)